Source/Core/DiscIO/NANDImporter.cpp

```diff
@@ -51,7 +52,8 @@ bool NANDImporter::ReadNANDBin(const std::string& path_to_bin)
   File::IOFile file(path_to_bin, "rb");
   if (file.GetSize() != NAND_BIN_SIZE)
   {
-    PanicAlertT("This file does not look like a BootMii NAND backup. (0x%lx does not equal 0x%zx)",
+    PanicAlertT("This file does not look like a BootMii NAND backup. (0%" PRIx64
```

Comment:
This only appends `0x` to the result if it's a non-zero result, which is rarely what you want for a consistent formatting.